src/components/dip-switch/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
```

Comment:
Btw. we can also nest components in the directory. So, `dip-switches` could be a subfolder of `board`.

Reply:
It's just a hint that we it could also be nested. It's also okay for me this way. I already approved 👍 .

Reply:
Changed to a nested component.